pkgs/development/python-modules/import-expression/default.nix

```
@@ -0,0 +1,45 @@
+{ lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytest,
+  pytest-cov,
+  astunparse,
+}:
+buildPythonPackage rec {
+  pname = "import-expression";
+  version = "1.1.4";
+  src = fetchFromGitHub {
+    owner = "ioistired";
+    repo = "import-expression-parser";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-mll2NePB7fthzltLOk6D9BgaDpH6GaW4psqcGun/0qM=";
+  };
+  pythonImportsCheck = [
+    "import_expression"
+    "import_expression._codec"
+  ];
+  propagatedBuildInputs = [ astunparse ];
+  nativeCheckInputs = [ pytest pytest-cov ];
```

```suggestion
  nativeCheckInputs = [ pytestCheckHook ];
```
Please use pytestCheckHook instead to avoid the redundant code.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook

---

Will do, thank you.